src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -73,9 +73,19 @@ public void Initialize(ExtensionConfigContext context)
 
             StaticServiceHubContextStore.ServiceManagerStore = new ServiceManagerStore(options.AzureSignalRServiceTransportType, configuration, loggerFactory);
 
-            var url = context.GetWebhookHandler();
-            logger.LogInformation($"Registered SignalR trigger Endpoint = {url?.GetLeftPart(UriPartial.Path)}");
-
+            bool triggerEnabled = false;
+            try
+            {
+                var url = context.GetWebhookHandler();
+                logger.LogInformation($"Registered SignalR trigger Endpoint = {url?.GetLeftPart(UriPartial.Path)}");
+                triggerEnabled = true;
+            }
+            catch
+            {
+                logger.LogInformation("SignalR input and output bindings are enabled. " +
+                                      "To use the SignalR trigger, please set 'AzureWebJobsStorage' to a valid connection string.");
```

Comment:
I think Log.Info only shows the information about important steps user hit. It seems not very effective to show suggestions in it. And if user doesn't use it correctly, the exception log throws is good enough to provide suggestions.

Reply:
I save the exception and throw it only when customer try to use `SignalRTrigger`.